### ISSUE TITLE
[BUG]: Running AP in the docker failed with extract metadata -127

### ISSUE THREAD

**Issue by glani** [activepieces/activepieces#3645]
**Describe the bug**
cross-env is not found

{"level":50,"time":"2024-01-15T19:04:35.740Z","pid":7,"hostname":"1b75441810e2","err":{"type":"Error","message":"Command failed with code 127: cd /usr/src/app/dist/cache/sandbox/0 && cross-env AP_PIECES_SOURCE=CLOUD_AND_DB NODE_OPTIONS=--enable-source-maps && \"/usr/local/bin/node\" main.js EXTRACT_PIECE_METADATA","stack":"Error: Command failed with code 127: cd /usr/src/app/dist/cache/sandbox/0 && cross-env AP_PIECES_SOURCE=CLOUD_AND_DB NODE_OPTIONS=--enable-source-maps && \"/usr/local/bin/node\" main.js EXTRACT_PIECE_METADATA\n    at FileSandbox.<anonymous> (webpack:///src/app/workers/sandbox/file-sandbox.ts:110:28)\n    at Generator.next (<anonymous>)\n    at /usr/src/app/dist/packages/backend/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/usr/src/app/dist/packages/backend/node_modules/tslib/tslib.js:165:16)\n    at code (webpack:///src/app/workers/sandbox/file-sandbox.ts:108:56)\n    at ChildProcess.emit (node:events:514:28)\n    at ChildProcess.emit (node:domain:489:12)\n    at maybeClose (node:internal/child_process:1091:16)\n    at Socket.<anonymous> (node:internal/child_process:449:11)"},"msg":"[PieceService#add]"}

**To Reproduce**
Prepare a new server with docker-compose
Restore the database with existed history and custom pieces from dump and start installing custom pieces.

**Expected behavior**
no error

**Additional context**
Running 
```
    export CONTAINER_ID=`docker ps -aqf name=test_ap`
    docker container exec -it $CONTAINER_ID /bin/sh
   npm i -g cross-env
```

solves the issue

**Comment by abuaboud** [comment]
Hi @glani 

Is this version 0.17.0? because it suppose to be already fixed there.

https://github.com/activepieces/activepieces/blob/main/Dockerfile#L26

**Comment by glani** [comment]
yes old one: self-activepieces-aarch64:0.16.0.0009671-0.3.1-7839-g9217a3994

I think master from Sundayor Saturday merged on my branch

**Comment by abuaboud** [comment]
Hi @glani,

I believe the problem has been resolved in version 0.17.0. I am closing this issue as fixed. Feel free to reopen if you have any further questions.
